### PR TITLE
Fix multiple mistakes and warnings

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

File: Emby.Server.Implementations/LiveTv/Listings/SchedulesDirect.cs [hunk 1]
```diff
@@ -152,22 +155,22 @@ public async Task<IEnumerable<ProgramInfo>> GetProgramsAsync(ListingsProviderInf
                                 var imagesWithText = allImages.Where(i => string.Equals(i.text, "yes", StringComparison.OrdinalIgnoreCase));
                                 var imagesWithoutText = allImages.Where(i => string.Equals(i.text, "no", StringComparison.OrdinalIgnoreCase));
 
-                                const double desiredAspect = 0.666666667;
+                                const double DesiredAspect = 0.666666667;
```
Comment (JustAMan) [inline Emby.Server.Implementations/LiveTv/Listings/SchedulesDirect.cs]: can we do it like this for clarity?

```suggestion
                                const double DesiredAspect = 2./3.;
```

File: Emby.Server.Implementations/LiveTv/Listings/SchedulesDirect.cs [hunk 2]
```diff
@@ -152,22 +155,22 @@ public async Task<IEnumerable<ProgramInfo>> GetProgramsAsync(ListingsProviderInf
...
 
-                                programEntry.primaryImage = GetProgramImage(ApiUrl, imagesWithText, true, desiredAspect) ??
-                                    GetProgramImage(ApiUrl, allImages, true, desiredAspect);
+                                programEntry.primaryImage = GetProgramImage(ApiUrl, imagesWithText, true, DesiredAspect) ??
+                                    GetProgramImage(ApiUrl, allImages, true, DesiredAspect);
 
-                                const double wideAspect = 1.77777778;
+                                const double WideAspect = 1.77777778;
```
Comment (JustAMan) [inline Emby.Server.Implementations/LiveTv/Listings/SchedulesDirect.cs]:
```suggestion
                                const double WideAspect = 16./9.;
```

File: MediaBrowser.Controller/Entities/BaseItem.cs [hunk 3]
```diff
@@ -2085,14 +2085,17 @@ public void AddStudio(string name)
...
+                if (curLen == 0)
                 {
                     Studios = new[] { name };
                 }
                 else
                 {
-                    var list =
-                    Studios = current.Concat(new[] { name }).ToArray();
+                    var newArr = new string[curLen + 1];
+                    current.CopyTo(newArr, 0);
```
Comment (JustAMan) [inline MediaBrowser.Controller/Entities/BaseItem.cs]: can you explain what was wrong with `current.Concat`?

File: MediaBrowser.XbmcMetadata/Savers/EpisodeNfoSaver.cs [hunk 4]
```diff
@@ -31,7 +31,7 @@ protected override string GetRootElementName(BaseItem item)
 
         /// <inheritdoc />
         public override bool IsEnabledFor(BaseItem item, ItemUpdateType updateType)
-            => !item.SupportsLocalMetadata && item is Episode && updateType >= MinimumUpdateType;
+            => item.SupportsLocalMetadata && item is Episode && updateType >= MinimumUpdateType;
```
Comment (JustAMan) [inline MediaBrowser.XbmcMetadata/Savers/EpisodeNfoSaver.cs]: LOL! xD

File: MediaBrowser.Api/UserLibrary/ItemsService.cs [hunk 5]
```diff
@@ -455,9 +455,7 @@ private InternalItemsQuery GetItemsQuery(GetItems request, DtoOptions dtoOptions
                         IncludeItemTypes = new[] { typeof(MusicAlbum).Name },
                         Name = i,
                         Limit = 1
-
-                    }).Select(albumId => albumId);
```
[the remark this answers is not in the document]
Comment (Bond-009) [reply MediaBrowser.Api/UserLibrary/ItemsService.cs]: Why? The select did nothing.

File: MediaBrowser.Controller/Entities/BaseItem.cs (hunk 3 above)
Comment (Bond-009) [reply MediaBrowser.Controller/Entities/BaseItem.cs]: It’s way slower and causes more allocations.

File: MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs [hunk 6]
```diff
@@ -1252,9 +1252,6 @@ private static int ScaleBitrate(int bitrate, string inputVideoCodec, string outp
         {
             if (request.AudioBitRate.HasValue)
             {
-                // Make sure we don't request a bitrate higher than the source
-                var currentBitrate = audioStream == null ? request.AudioBitRate.Value : audioStream.BitRate ?? request.AudioBitRate.Value;
-
                 // Don't encode any higher than this
                 return Math.Min(384000, request.AudioBitRate.Value);
```
[the remark this answers is not in the document]
Comment (Bond-009) [reply MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs]: 🤷‍♂️ I just removed an unused variable without changing logic.

File: MediaBrowser.Api/UserLibrary/ItemsService.cs (hunk 5 above)
[the remark this answers is not in the document]
Comment (JustAMan) [reply MediaBrowser.Api/UserLibrary/ItemsService.cs]: I thought it extracted one column from the "table", am I reading the code wrong?

File: MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs (hunk 6 above)
[the remark this answers is not in the document]
Comment (JustAMan) [reply MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs]: For me previously that looked like an error of not using the variable which the code expected to use... though if we're re-encoding from lossy format to lossy one (and we're most likely doing it that way) original idea of utilizing source bitrate is meaningless, so let's keep this simpler version.

**Pull request merged**